docs/includes/topic_f5lbaas-l7_content_switching.rst

```diff
@@ -218,9 +220,9 @@ Configuration
 Further Reading
 ---------------
 
```

Comment:
Yeah, I started out with that and it fails with the same error. I thought the idea is to reference a footnote in the text somewhere, like [#f1]_, then to dereference it at the bottom.